Frends.OAuth.CreateJWTToken/Frends.OAuth.CreateJWTToken/CreateJWTToken.cs

```diff
@@ -24,6 +24,7 @@ public static TokenResult CreateJWTToken(Input input)
         SigningCredentials signingCredentials;
         bool isSymmetric = input.SigningAlgorithm.ToString().StartsWith("HS");
         using var rsa = RSA.Create();
+        using var ecdsa = ECDsa.Create();
```

Comment:
Do you need to create these if using "HS" starting SigningAlgorithm? Should these be assigned in their if statements?

Reply:
Yes you have. SigningCredentials-object is passed to CreateToken-method at line 51. If using-statement would be inside if-statement, ECDsa-object will be disposed when execution goes out of the if-clause. You can verify this by moving the using statement inside if-clause and running unit tests.